olm-catalog/serverless-operator/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM scratch
+
+COPY manifests /manifests
+COPY metadata/annotations.yaml /metadata/annotations.yaml
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=serverless-operator
+LABEL operators.operatorframework.io.bundle.channel.default.v1="4.3"
+LABEL operators.operatorframework.io.bundle.channels.v1="4.3,4.4,4.5"
```

Comment:
Oh. Are we releasing into the 4.5 channel? Technically, I don't think it's possible because we already have a newer version there (1.8.0, 1.9.0) and 1.7.3 does not "replace" 1.9.0 in the CSV.

Reply:
I guess we will see if the skipRange alows for this setup. There's already 1.8.0 which replaces 1.7.2. It may work. Not sure